aws/sdk/aws-models/ebs-tests.smithy

```diff
@@ -0,0 +1,35 @@
+$version: "1.0"
+
+namespace com.amazonaws.ebs
+
+use smithy.test#httpRequestTests
+use aws.protocols#restJson1
+use com.amazonaws.ebs#PutSnapshotBlock
+apply PutSnapshotBlock @httpRequestTests([
```

Comment:
Can we make a more minimal, focused, test for this to avoid pulling in an additional service? Perhaps a unit test for SigV4AuthDecorator?